File: forum-angular/src/app/DTO/Comment.ts
import { User } from './User';
export class Comment
{

     public  Id :number
     public  postId :number
     public  userId :number

     public  content :string
     public  img :string

     public  date :string



     public  comment :Comment


     public  user :User

     constructor(){

     }


}

File: forum-master/FORUM/Models/Comment.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace FORUM.Models
{
    public class Comment
    {

         [Key]
         public int Id { get; set; }
    
         public Post post { get; set; }
        public int userId { get; set; }
        public int postId { get; set; }


         public string content { get; set; }
         public string img { get; set; }

         public string date { get; set; }


        

         public User user { get; set; }


    }
}